Docs/API.md

```diff
@@ -837,6 +839,8 @@ __Parameters__
 | ``objectName``  | _string_  | Object name in the bucket |
 | ``fileName``  | _string_  | File to upload |
 | ``contentType``  | _string_ | Content type of the file. Defaults to " |
+| ``metadata``  | _Dictionary<string,string>_ | Dictionary of meta data headers and their values.Defaults to null.|
```

Comment:
The description for `metadata` could be,

Object metadata to be stored. Defaults to _null_.